biome.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.3.3/schema.json",
+  "formatter": {
+    "include": [],
+    "indentSize": 2,
+    "indentStyle": "space",
+    "indentWidth": 120,
```

I think there is a configuration error here.
`indentWidth` replaces the deprecated `indentSize` option.

```suggestion
    "indentStyle": "space",
    "indentWidth": 2,
```

---

Ah, thanks!